packages/fetch-http-handler/src/stream-collector.ts

```diff
@@ -44,8 +44,11 @@ function readToBase64(blob: Blob): Promise<string> {
       if (reader.readyState !== 2) {
         return reject(new Error("Reader aborted too early"));
       }
-      const result = reader.result as string;
-      const dataOffset = result.indexOf(",") + 1;
+      const result = (reader.result as string) || "";
```

Comment:
```suggestion
      const result = (reader.result ?? "") as string;
```

TypeScript playgrounds:
* [Calling as string on result](https://www.typescriptlang.org/play?ts=3.8.3#code/C4TwDgpgBAThDOBXANsAKuaBeK9gwEsA7AcygB8oiVkBuAKHoGMB7IvKVogNwhnRYBlfMTI4AFMBYAhCAGE2vfhAAmALlgIU6TAEoNeQqShYAfFCmyFPPsFVQAhvFwjj5SgCIPDZottohVxJxD0MPXQYuJQFhI2DqZGQI+iA)
* [Calling as string on result ?? ""](https://www.typescriptlang.org/play?ts=3.8.3#code/C4TwDgpgBAThDOBXANsAKuaBeK9gwEsA7AcygB8oiVkBuAKHoGMB7IvKVogNwhnRYBlfMTI4AFMBYAhCAGE2vfhAAmALlgIU6TAEoNeQqShYAfFCmyFPPsFVQA-A6gAiFw2aLbaISNLiXQxddBi4lAWEjEnFqZGQQ+iA)